### DISCUSSION TITLE
End the calculation directly

### DISCUSSION THREAD

**Discussion by xiaoheimai95** [CURENT/andes#470] [Q&A] [answered]
Dear Dr. Cui and Jinning,
I want to know if there is a way to set it up so that when the calculation time exceeds a certain range, the calculation program is simply terminated.

Best wishes!

**Comment by jinningwang** [comment, marked as answer]
I'm afraid that we don't have such a function. ANDES has the real-time simulation feature defined by configuration ``kqrt``, https://docs.andes.app/en/latest/configref.html?highlight=real%20time#tds, but this feature seems not that helpful for your needs.

Python's built-in multi-processing feature might be used for your implementation, where your main program and ANDES program can be set as two separate processes.

**Reply by xiaoheimai95** [reply]
Thanks a lot!
